src/components/whtr/styles/whtr-chart-axes.module.scss

```diff
@@ -0,0 +1,45 @@
+@import './whtr-chart.config.module.scss';
+@import '../../styles/mixins.module.scss';
+
+$axis-offset: 5px;
+
+@mixin chart-marker-size($size) {
+  height: $size + $axis-offset;
+}
+
+.nhsuk-whtr-axis-start-marker {
+  position: absolute;
+  bottom: -5px;
+  margin-left: -0.5px;
+  box-shadow: 0px 0px white inset, 1px 0px white, -1px 0px white;
+  z-index: 4;
+  @include chart-marker-size($whtr-graph-height-desktop);
+  @include for-mobile {
+    @include chart-marker-size($whtr-graph-height-mobile);
+  }
```

Comment:
is this quite a common set of css for the marker? If so could we turn it into a mixin and use throughout

Reply:
yea it is. ok will have a look

Reply:
resolved